modules/widgets/facebook-likebox.php

```diff
@@ -207,17 +211,36 @@ function form( $instance ) {
 			</label>
 		</p>
 
+		<p>
+			<label for="<?php echo esc_attr( $this->get_field_id( 'hide_cta' ) ); ?>">
+				<input type="checkbox" name="<?php echo esc_attr( $this->get_field_name( 'hide_cta' ) ); ?>" id="<?php echo esc_attr( $this->get_field_id( 'hide_cta' ) ); ?>" <?php checked( $like_args['hide_cta'] ); ?> />
+				<?php _e( 'Hide Call To Action Button', 'jetpack' ); ?>
+				<br />
+				<small><?php _e( 'Hides the custom call to action button (if available).', 'jetpack' ); ?></small>
```

Comment:
I think this may not mean much for most people. "Call To Action" is a bit of a jargon term, don't you think? It makes me wonder if we should really add those options to the widget settings. Maybe we could instead just have a filter around the Facebook timeline's HTML container, so folks could edit it with a functionality plugin. What do you think?

Reply:
I agree. That's a good idea. I'll do that!

Reply:
This is done.